pkg/reconciler/cluster/controller.go

```diff
@@ -206,20 +270,48 @@ func (c *Controller) deletedCluster(obj interface{}) {
 }
 
 func RegisterClusterCRD(cfg *rest.Config) error {
-	bytes, err := ioutil.ReadFile("config/cluster.example.dev_clusters.yaml")
-
 	crdClient := apiextensionsv1client.NewForConfigOrDie(cfg)
 
-	crd := &apiextensionsv1.CustomResourceDefinition{}
-	err = yaml.Unmarshal(bytes, crd)
+	files, err := ioutil.ReadDir("config/")
 	if err != nil {
 		return err
 	}
+	for _, file := range files {
+		if !strings.HasSuffix(file.Name(), "yaml") {
+			continue
+		}
+		bytes, err := ioutil.ReadFile("config/" + file.Name())
+		crd := &apiextensionsv1.CustomResourceDefinition{}
+		err = yaml.Unmarshal(bytes, crd)
+		if err != nil {
+			return err
+		}
 
-	_, err = crdClient.CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
-	if err != nil && !k8serorrs.IsAlreadyExists(err) {
-		return err
+		_, err = crdClient.CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
+		if k8serorrs.IsAlreadyExists(err) {
+			existingCRD, err := crdClient.CustomResourceDefinitions().Get(context.TODO(), crd.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			crd.ResourceVersion = existingCRD.ResourceVersion
+			crdClient.CustomResourceDefinitions().Update(context.TODO(), crd, metav1.UpdateOptions{})
+		} else if err != nil {
+			return err
+		}
 	}
 
 	return nil
 }
+
+var clusterKind string = reflect.TypeOf(clusterv1alpha1.Cluster{}).Name()
+
+func ClusterAsOwnerReference(obj *clusterv1alpha1.Cluster, controller bool) metav1.OwnerReference {
+	isController := controller
+	return metav1.OwnerReference{
+		APIVersion: apiresourcev1alpha1.SchemeGroupVersion.String(),
+		Kind:       clusterKind,
+		Name:       obj.Name,
+		UID:        obj.UID,
+		Controller: &isController,
```

Comment:
I think you can just do `&controller` instead of involving `isController`

Reply:
Fixed in commit 6a4dcd07a4a268ef7dd0158950e2a1c8452b9eb3